guides/source/getting_started.md

```diff
@@ -1985,7 +1987,6 @@ Article model, `app/models/article.rb`, as follows:
 ```ruby
 class Article < ApplicationRecord
   include Visible
-
```

Comment:
When the line `include Visible` was first introduced in [Section 8.3](https://guides.rubyonrails.org/getting_started.html#using-concerns), there was no line between `include Visible` and `has_many :comments`.

Reply:
Can we add the line in the previous section. That line should be there.

Reply:
Yes, I will do that. Thank you.